generator.json

```diff
@@ -641,7 +643,8 @@
                     "EGLConfig": "nint",
                     "EGLContext": "nint",
                     "GLXFBConfig": "nint",
-                    "VkFormat": "uint"
+                    "VkFormat": "uint",
```

Comment:
Maybe missing context, but that does not seem right at all?

Reply:
It is.